app/views/shared/_tea_times.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="tt-partial">
+  <% if host ||= nil %>
+    <h3>
+      <!--TODO: Remove after Forbes/Inc-->
+      <%= 'Join for an upcoming tea time with #{host.name}' %>
+    </h3>
+  <% end %>
+  <div class="tt-set-container">
+    <% tea_times.each do |tt| %>
+      <%= render partial: 'shared/tea_time_row', locals: {tea_time: tt} %>
+    <% end %>
+  </div>
+  <% if (host ||= nil) %>
```

Comment:
This is actually not something I wrote. This was just converted from the haml version. It was a dumb conditional put in place to accommodate the tea time partial that was placed at the bottom of host profiles